dev-qt/qtwebkit/qtwebkit-5.212.0_pre20180120.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+COMMIT=72cfbd7664f21fcc0e62b869a6b01bf73eb5e7da
+CMAKE_MAKEFILE_GENERATOR="ninja"
+PYTHON_COMPAT=( python2_7 )
+QT_MIN_VER="5.9.1:5"
+USE_RUBY="ruby23 ruby24 ruby25"
+inherit check-reqs cmake-utils flag-o-matic python-any-r1 qmake-utils ruby-single toolchain-funcs
+
+DESCRIPTION="WebKit rendering library for the Qt5 framework (deprecated)"
+HOMEPAGE="https://www.qt.io/"
+SRC_URI="http://code.qt.io/cgit/qt/${PN}.git/snapshot/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+if [[ ${PV} != *9999* ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+fi
+
+SLOT=5/5.212
+LICENSE="BSD LGPL-2+"
+IUSE="+geolocation gles2 +gstreamer +hyphen +jit multimedia nsplugin opengl orientation +printsupport qml webp X"
+
+REQUIRED_USE="
+	nsplugin? ( X )
+	qml? ( opengl )
+	?? ( gstreamer multimedia )
+"
+
+# Dependencies found at Source/cmake/OptionsQt.cmake
+RDEPEND="
+	dev-db/sqlite:3
+	dev-libs/icu:=
+	dev-libs/libxml2:2
+	dev-libs/libxslt
+	>=dev-qt/qtcore-${QT_MIN_VER}
+	>=dev-qt/qtgui-${QT_MIN_VER}
+	>=dev-qt/qtnetwork-${QT_MIN_VER}
+	>=dev-qt/qtwidgets-${QT_MIN_VER}
+	media-libs/libpng:0=
+	virtual/jpeg:0
+	geolocation? ( >=dev-qt/qtpositioning-${QT_MIN_VER} )
+	gstreamer? (
+		dev-libs/glib:2
+		media-libs/gstreamer:1.0
+		media-libs/gst-plugins-base:1.0
+		media-libs/gst-plugins-bad:1.0
+	)
+	hyphen? ( dev-libs/hyphen )
+	multimedia? ( >=dev-qt/qtmultimedia-${QT_MIN_VER}[widgets] )
+	opengl? (
+		>=dev-qt/qtgui-${QT_MIN_VER}[gles2=]
+		>=dev-qt/qtopengl-${QT_MIN_VER}
+	)
+	orientation? ( >=dev-qt/qtsensors-${QT_MIN_VER} )
+	printsupport? ( >=dev-qt/qtprintsupport-${QT_MIN_VER} )
+	qml? (
+		>=dev-qt/qtdeclarative-${QT_MIN_VER}
+		>=dev-qt/qtwebchannel-${QT_MIN_VER}[qml]
+	)
+	webp? ( media-libs/libwebp:0= )
+	X? (
+		x11-libs/libX11
+		x11-libs/libXcomposite
+		x11-libs/libXrender
+	)
+"
+
+# Need real bison, not yacc
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	${RUBY_DEPS}
+	dev-lang/perl
+	dev-util/gperf
+	>=sys-devel/bison-2.4.3
+	sys-devel/flex
+	virtual/pkgconfig
+"
+
+CHECKREQS_DISK_BUILD="1G" # Debug build requires much more see bug #417307
```

Comment:
I'm confused, if debug builds require much more, why are we checking for only 1G with `-g`?

This should be bumped to something like 16G according to the referenced bug, although the comments there are pretty old so things could have changed.

Reply:
the original ebuild was based off a much, much outdated state. this is too low for sure.